cypress/e2e/components/header.cy.js

```diff
@@ -49,27 +44,39 @@ describe('check headerimage and content', () => {
     });
     context('check topheader images', () => {
         it('check header image', () => {
-            cy.inHeader(() => {
-                const headerImage = cy.get('img.topHeader__image');
-                headerImage
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img')
                     .should('be.visible')
                     .should('have.attr', 'src', 'img/header.jpg');
             });
         });
         it('check header logo', () => {
-            cy.inHeader(() => {
-                const logoArea = cy.get('div.topHeader__logoArea');
-                logoArea.should('be.visible');
-                logoArea.within(() => {
-                    cy.get('svg');
-                    cy.get('p').should('have.length', 2);
-                });
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get(componentSelectors.logoArea)
+                    .should('be.visible')
+                    .within(() => {
+                        cy.get('svg').should('be.visible');
+                        cy.get('p').should('have.length', 2);
+                    });
             });
         });
         it('check header and logo sizes', () => {
-            cy.inHeader(() => {
-                const headerImage = cy.get('img.topHeader__image');
-                headerImage.invoke('width').should('be.lessThan', 1920);
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img').invoke('width').should('be.lessThan', 1920);
+            });
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get(componentSelectors.logoArea).should(
+                    'have.css',
+                    'position',
+                    'absolute',
+                );
+            });
+        });
+        it('check header and logo sizes in smaller breakpoint', () => {
+            cy.viewport(1200, 768);
+            cy.get(componentSelectors.topHeader).within(() => {
+                cy.get('img').invoke('width').should('be.lessThan', 900);
+                cy.get(componentSelectors.logoArea).should('have.css', 'right');
```

Comment:
```suggestion
                cy.get(componentSelectors.logoArea).should('have.css', 'right', '0');
```
soll ja 0 sein diese rechte diese

Reply:
behoben